pkg/app/web/strip-imports.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
I think we could remove this line 👀

Reply:
Ah, it throws an error by ESLint currently.
But I think it does not need this rule for `.js` files.
I will change `.eslintrc` to exclude `.js` files from that rule later. 
Thank you!

Reply:
I see, thank you :pray: